app/routes/app/ecosystem/metrics.tsx

```diff
@@ -102,6 +102,10 @@ function StatsCard({
   description?: string;
   icon?: React.ReactNode;
 }) {
+  if (typeof figure === "number") {
```

Comment:
Can just make figure a number and not need the typeof check. It should be a number that we get back from whatever backend system.

Reply:
The bottom right card (rate of decay) isn't a number. Which is why it allows a string or number

Reply:
ah I see. In that case I would probably opt to make figure a string, but feeling kinda nitpicky at this point because its also nice to have a component that can handle both types.